modules/apps/27-interchain-accounts/module.go

```diff
@@ -243,6 +250,11 @@ func (am AppModule) OnAcknowledgementPacket(
 		return err
 	}
 
+	if packet.Sequence == 1 && ack.Success() {
+		// pass interchain account address into base applications
+		// connected to the middleware
+	}
```

Comment:
this information shouldn't be set in state since we could have domain conflicts (this module acts as a host and controller in some cases). It's usefulness is passing this information to helper modules, but even so we will likely end up using the version to convey this information